Framework/Algorithms/src/PDCalibration.cpp

```diff
@@ -158,41 +177,35 @@ const std::string PDCalibration::summary() const {
  */
 void PDCalibration::init() {
   declareProperty(Kernel::make_unique<WorkspaceProperty<MatrixWorkspace>>(
-                      "SignalWorkspace", "", Direction::InOut),
+                      "InputWorkspace", "", Direction::InOut),
```

Comment:
The usage example should be update dated for [PDCalibration](http://docs.mantidproject.org/nightly/algorithms/PDCalibration-v1.html#usage). I would be surprised with all these changes if the FitPeaks docs didn't also need to be updated.